android/src/main/java/com/tailscale/ipn/ui/view/SettingsView.kt

```diff
@@ -110,7 +127,14 @@ object Setting {
               style = MaterialTheme.typography.bodyMedium,
               color = if (destructive) MaterialTheme.colorScheme.error else Color.Unspecified)
         },
-    )
+        supportingContent = {
```

Comment:
Please change this to the below to avoid having empty supporting content.

```
supportingContent = subtitle?.let {
            Text(
                it,
                style = MaterialTheme.typography.bodySmall,
                color = MaterialTheme.colorScheme.onSurfaceVariant)
        })
```

Reply:
That needs to be a closure, but yes, done!